modules/alloy/src/test/ts/atomic/position/view/BounderDeterminePositionTest.ts

```diff
@@ -0,0 +1,77 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { assert } from 'chai';
+
+import * as Boxes from 'ephox/alloy/alien/Boxes';
+import * as Bounder from 'ephox/alloy/positioning/view/Bounder';
+
+describe('BounderDeterminePositionTest', () => {
+  it('inside the bounds', () => {
```

Comment:
Should these have Jira numbers / TBAs?

Reply:
Yeah, this should have a JIRA number thanks.

Reply:
Done